tests/integration.coffee

```diff
@@ -48,6 +48,9 @@ if not _.every [
 	console.error('Missing environment credentials')
 	process.exit(1)
 
+buildDeviceDashboardUrl = (appId, deviceId) ->
```

Comment:
@pcarranzav yes, thank you for pointing out it makes perfect sense. I've updated the PR by adding a `resin.models.device.getDashboardUrl()` method cc @jviotti